Tests/WorkspaceTests/SourceControlPackageContainerTests.swift

```diff
@@ -234,7 +234,7 @@ class SourceControlPackageContainerTests: XCTestCase {
     func testVersions() throws {
         let fs = InMemoryFileSystem()
 
-        let repoPath = AbsolutePath.root
+        let repoPath = AbsolutePath.root.appending("SourceCache")
```

Comment:
question: Should we use `withTemporaryDirectory` here instead?  Untested e.g


```
func testVersions() throws {
    try withTemporaryDirectory(removeTreeOnDeinit: true) { repoPath in
        let filePath = repoPath.appending("Package.swift")

        //....

    }
}
```

This way, we won't risk corrupting the host drive folder.

Reply:
We can do that in place of the `InMemoryFileSystem`. I think that the test issues do often arise from the use of `InMemoryFileSystem`.